src/reconciler.ts

```diff
@@ -1,20 +1,28 @@
+import {ReactNode} from 'react';
 import {
 	unstable_scheduleCallback as schedulePassiveEffects,
 	unstable_cancelCallback as cancelPassiveEffects
 } from 'scheduler';
-import ReactReconciler from 'react-reconciler';
+import createReconciler, {HostConfig} from 'react-reconciler';
 import {
 	createNode,
 	createTextNode,
 	appendChildNode,
 	insertBeforeNode,
 	removeChildNode,
-	setAttribute
+	setAttribute,
+	NodeNames,
+	DOMNode
 } from './dom';
 
 const NO_CONTEXT = true;
 
-const hostConfig = {
+interface Props {
+	children: ReactNode;
+}
+
+const hostConfig: HostConfig<NodeNames, Props, DOMNode, any, any, any, any, any, any, any, any, any> = {
```

Comment:
Can we use `unknown` instead of `any` here?

Reply:
Done